packages/block-library/src/list/editor.scss

```
@@ -3,3 +3,21 @@
 	padding-left: 1.3em;
 	margin-left: 1.3em;
 }
+
+.editor-styles-wrapper .block-library-list ol {
+	&.ol-type-is-1 {
+		list-style: decial;
```

Looks like a typo here

```suggestion
		list-style: decimal;
```

---

Fixed